src/crawlee/http_clients/_httpx.py

```diff
@@ -222,13 +229,11 @@ def _get_client(self, proxy_url: str | None) -> httpx.AsyncClient:
         if proxy_url not in self._client_by_proxy_url:
             # Prepare a default kwargs for the new client.
             kwargs: dict[str, Any] = {
-                'transport': _HttpxTransport(
-                    proxy=proxy_url, http1=self._http1, http2=self._http2, verify=self._verify
-                ),
```

Comment:
I checked now and it looks like the `transport` doesn't need to know the proxy URL since proxying is handled by the `httpx.AsyncClient` (it creates a separate transport for that).